app/build.gradle

```diff
@@ -54,11 +57,16 @@ dependencies {
     implementation "androidx.fragment:fragment-ktx:1.5.5"
     debugImplementation("androidx.fragment:fragment-testing:1.5.5")
 
-    // Real time DB (check the following link to add any other dependency:
+    // Real time DB + google authentificator (check the following link to add any other dependency:
     // https://firebase.google.com/docs/android/setup#available-libraries
     implementation platform('com.google.firebase:firebase-bom:31.2.3')
     implementation 'com.google.firebase:firebase-analytics-ktx'
     implementation 'com.google.firebase:firebase-database-ktx'
+    implementation 'com.google.firebase:firebase-auth-ktx:21.1.0'
+
+    //coroutines
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.7.0-Beta")
```

Comment:
I think this import is not useful, coroutines seems to already be available in android environment without it.

Reply:
Indeed, I'll delete this import, thanks!